.github/workflows/frontend.yml

```
@@ -50,15 +50,28 @@ jobs:
-        run: npm run lint
+      - name: Auto-fix Lint (ESLint)
+        continue-on-error: true
+        run: npx ng lint --fix
 
       - name: Custom Lint
+        continue-on-error: true
         run: npm run lint:custom
 
+      - name: Commit and Push Auto-fixes
+        if: always() && github.event.pull_request.head.repo.full_name == github.repository
+        run: |
```

actionlint (1.7.12)

[error] "github.event.pull_request.head.ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)